docs/en/macros/augmentation-args.md

```diff
@@ -13,7 +13,7 @@
 | `bgr`             | `float` | `0.0`         | `0.0 - 1.0`   | Flips the image channels from RGB to BGR with the specified probability, useful for increasing robustness to incorrect channel ordering.                                  |
 | `mosaic`          | `float` | `1.0`         | `0.0 - 1.0`   | Combines four training images into one, simulating different scene compositions and object interactions. Highly effective for complex scene understanding.                |
 | `mixup`           | `float` | `0.0`         | `0.0 - 1.0`   | Blends two images and their labels, creating a composite image. Enhances the model's ability to generalize by introducing label noise and visual variability.             |
-| `copy_paste`      | `float` | `0.0`         | `0.0 - 1.0`   | Copies objects from one image and pastes them onto another, useful for increasing object instances and learning object occlusion.                                         |
+| `copy_paste`      | `float` | `0.0`         | `0.0 - 1.0`   | Copies and pastes objects across images, useful for increasing object instances and learning object occlusion. Applies only to segmentation training.                     |
```

Comment:
it's not segmentation training only, we could use it for object detection but it does need segment labels.

Reply:
@Laughing-q Ah ok. That makes sense.